bandit/src/query-lambda/queryBigQueryData.ts

```diff
@@ -0,0 +1,119 @@
+import {format, subDays} from "date-fns";
+import type {Test} from "../lib/models";
+import {Query} from "../lib/query";
+
+
+const ANNUALISED_VALUE_CAP = 250;
+export const buildQuery = (
+	test: Test,
+	stage: "CODE" | "PROD",
```

Comment:
How about using the `stage` parameter for the query, and outside of this function we can hardcode passing in `PROD`, with a comment explaining why we use it for both environments?
E.g. `FROM datatech-platform-${stage.toLowerCase()}.datalake.fact_acquisition_event`